console/src/components/Services/Common/ComponentErrorBoundary.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component, ReactNode } from 'react';
+import globals from '../../../Globals';
+import styles from '../../Error/ErrorPage.scss';
+
+const errorImageSrc = `${globals.assetsPath}/common/img/hasura_icon_green.svg`;
+
+interface ErrorState {
+  error: Error;
+  hasError: boolean;
```

Comment:
It's redundant. You can specify `error` ad `error: Error | null` or `error?: Error`